keyboards/kinesis/keymaps/farmergreg/config.h

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2022 Gregory L. Dietsche <gregory.dietsche@cuw.edu>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+
+//#define MANUFACTURER    "Dietsche Labs"
```

Comment:
If you're not defining these, this file can be deleted.

Reply:
removed, thanks!